### ISSUE TITLE
LInk to Info on Apps

### ISSUE THREAD

**Issue by PaPaTheGMan** [shmykelsa/AAAD#54]
It would be helpful to have a link to the apps offered so a person could read more about them.  I'm not familiar with many of the app available for download.   Although a nice to have, the link doesn't have to be in the app, it could just be in the FAQ.

**Comment by shmykelsa** [comment]
Hi, 

that's an awesome feedback! I'll try to make this for one of the next releases!

**Comment by mclever** [comment]
Yeah I didn't read the press and hold part of the button until it was too late. For me the AA monitor doesn't work, so I'm out of my free download and it was useless. If the screen mirroring will work I would pay $20, so a 1 day trial of all apps and then pay to use after that would be the best thing for me.

**Comment by shmykelsa** [comment]
> Yeah I didn't read the press and hold part of the button until it was too late. For me the AA monitor doesn't work, so I'm out of my free download and it was useless. If the screen mirroring will work I would pay $20, so a 1 day trial of all apps and then pay to use after that would be the best thing for me.

Hey, if a mirroring app goes black you might want to give permission to cast screen on phone's screen.

**Comment by mclever** [comment]
> Hey, if a mirroring app goes black you might want to give permission to cast screen on phone's screen.

Enable wireless projection is enabled. Black screen on card, phone app just flashes. Assigned all perms app it asked for. Also confirmed no other perms could be assigned to app.

**Comment by shmykelsa** [comment]
Ouch no, wireless projection does not make any difference. You should see a dialog on the screen upon opening. Asks for screen permission.

**Comment by mclever** [comment]
> Ouch no, wireless projection does not make any difference. You should see a dialog on the screen upon opening. Asks for screen permission.

there were two permissions that it asked for and i allowed both. even tried to reboot, no change.
